### ISSUE TITLE
Add new menu entry Settings

### ISSUE THREAD

**Issue by internetmosquito** [wizbots/labtab#108]
Add a new entry in main menu, call it Settings, before the logout option

![screenshot_20190107-111340_labtab](https://user-images.githubusercontent.com/1500392/50762662-e25d2080-126d-11e9-80e9-b7e81a1b5bb3.jpg)

For now, only show Wizbots version, which is impossible to see right now

**Comment by internetmosquito** [comment]
@kapoorutd This ain't working, getting this error while trying to fire up application

```
error: resource string/wizbots_version (aka org.wizbots.labtab:string/wizbots_version) not found.
Message{kind=ERROR, text=error: resource string/wizbots_version (aka org.wizbots.labtab:string/wizbots_version) not found., sources=[/home/internetmosquito/git/labtab/app/src/main/res/layout/fragment_settings.xml:12], original message=, tool name=Optional.of(AAPT)}
```

Seems related with this in the fragment

```
    <org.wizbots.labtab.customview.TextViewCustom
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_marginLeft="@dimen/extra_small_margin"
        android:gravity="center|left"
        android:text="@string/wizbots_version"
        android:textColor="@color/purple"
        android:textSize="@dimen/text_size_small"
        app:font="@string/font_pt_sans_narrow_regular" />
```
Please fix

**Comment by internetmosquito** [comment]
So I added missing part to strings.xml

```
<string name="wizbots_version">LabTab version is v1.8.5</string>
```

But then firing up the application noticed this

![selection_796](https://user-images.githubusercontent.com/1500392/50777913-11d75180-129d-11e9-8c49-1d26c6e4bccb.png)

I could see a fragment txt_version right next to the one that gets the above string, if I changed the visibility to hidden

![selection_795](https://user-images.githubusercontent.com/1500392/50777955-27e51200-129d-11e9-817e-ace06fc77ad9.png)

It works fine

![selection_794](https://user-images.githubusercontent.com/1500392/50777972-2f0c2000-129d-11e9-9110-fa25302b75eb.png)

Why do we need that one in the first place?

**Comment by kapoorutd** [comment]
@internetmosquito txt_version is for version text and will not be hard coded, txt_version value will be given at run time, currently it is fetching value for app version name (that is 1.0, written in build.gradle) we can also create a constant for labtab tab version like base urls
![screenshot 2019-01-08 at 11 17 42 am](https://user-images.githubusercontent.com/6163203/50812016-0c2e4a00-1337-11e9-9b46-e76eb249b526.png)
and you have to change it every time whenever you update version name. 

<string name="wizbots_version">LabTab version is v1.8.5</string> is also works fine and you have to update version values in string file, but changing string.xml for every build is not consider as good practise.

According to Android docs we should increase app version name and version code in app build.gradle
![screenshot 2019-01-08 at 11 28 47 am](https://user-images.githubusercontent.com/6163203/50812409-a347d180-1338-11e9-93af-765b3254e0dd.png)
so we suggest whenever you create new build, change the app version name and version code in app level build.gradle and it will automatically reflect on setting page like below image
![screenshot 2019-01-08 at 11 32 21 am](https://user-images.githubusercontent.com/6163203/50812535-1a7d6580-1339-11e9-8fd7-034f8e3c46c3.png)

**Comment by internetmosquito** [comment]
@kapoorutd fine with me, then please update gradle file and let me know when it's ready

**Comment by kapoorutd** [comment]
@internetmosquito we implemented as requested, please test and close the issue if fixed

**Comment by kapoorutd** [comment]
@internetmosquito we implemented as requested, please test and close the issue if fixed

**Comment by internetmosquito** [comment]
This was released in 1.8.6
